test/uk/gov/hmrc/agentservicesaccount/controllers/ContactDetailsControllerSpec.scala

```diff
@@ -365,12 +365,12 @@ class ContactDetailsControllerSpec extends UnitSpec with Matchers with GuiceOneA
     }
   }
 
-  "GET /contact-details-submitted" should {
+  "GET /manage-account/contact-details/confirmation" should {
     "display the enter telephone number page" in new TestSetup {
```

Comment:
Looks like test description needs to be changed "display the enter telephone number page"

Reply:
Done. Thanks.